src/lib/ngsiNotify/senderThread.cpp

```diff
@@ -46,6 +47,12 @@ void* startSenderThread(void* p)
     char                portV[STRING_SIZE_FOR_INT];
     std::string         url;
 
+    if (params->content.size() > notifMaxSize)
```

Comment:
Different for other cases (in which we have transformed a check based in a static limit to a check based in a dynamic limit) this (and the other similar one in QueueWorkers.cpp) are new checks.

Probably documentation was misleading and what we refer to "Notification maximum size" at https://fiware-orion.readthedocs.io/en/master/user/known_limitations/index.html#notification-maximum-size was really meaning "outgoing message maximum limit".

Should we introduce new checks? Or just make dynamic the existing ones?

Reply:
Yes, the new "notification max size check" is better as it is "only for notifications" and that it it stopped before getting all  the way to httpRequestSend.

The existing one (inside httpRequestSendwithCurl) is already made dynamic. The third CLI option sets the limit.

It would not be bad to have a specific limit also for Response and Forward but please, not in this PR ! :)
The easiest way to implement this I think would be to inform httpRequestSend with the type of the message. The checks would be in one single place.

Create an issue?

Reply:
I think that just with making dynamic the existing limits is enough for this PR (my comment at https://github.com/telefonicaid/fiware-orion/issues/3492#issuecomment-494680647 was about it, but I'm afraid I did not express correctly, due to the misleading documentation we have for this). We have two limits, so two CLIs. This extra check could be removed. The addition of extra checks could be discussed in an additional issue.